qemu/tests/single_driver_install.py

```diff
@@ -172,6 +173,15 @@ def run(test, params, env):
         if not utils_misc.wait_for(lambda: not session.cmd_status(chk_cmd),
                                    600, 60, 10):
             test.fail("Failed to install driver '%s'" % driver_name)
+        if device_name[0:30] == "Red Hat VirtIO Ethernet Adapte":
+            ext_host = utils_net.get_ip_address_by_interface(ifname="%s" % params.get("netdst"))
+            test.log.info("ext_host of netkvm adapte is %s" % ext_host)
+            guest_ip = vm.get_address("nic2")
```

Comment:
Could you please tell why using 'nic2' here?  I thought it is similar with 'index2'. How about getting the ether name inside guest with some keywords, such as "Red Hat VirtIO Ethernet Adapte"

Reply:
And "nic2" has already been defined in our cfg file: [tp-qemu/blob/master/qemu/tests/cfg/single_driver_install.cfg#L56](https://github.com/autotest/tp-qemu/blob/master/qemu/tests/cfg/single_driver_install.cfg#L56).